apps/alerts/mix.exs

```diff
@@ -22,7 +22,7 @@ defmodule Alerts.Mixfile do
   # Type "mix help compile.app" for more information
   def application do
     [
-      applications: [:logger, :timex, :repo_cache, :con_cache, :tzdata, :util],
```

Comment:
Are the ones but `:logger` inferred and that's why they're removed from the list?

Reply:
Yep, they're part of the `deps` list so Elixir already knows we depend on them. `extra_applications` was added back in Elixir 1.4 (https://elixir-lang.org/blog/2017/01/05/elixir-v1-4-0-released/), but we never went back to update our configuration.